pkg/commands/start_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package commands
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestStart(t *testing.T) {
```

Comment:
For testing, I think it would be better if the test itself launched the host resolver (rather than relying on the Dockerfile to do it). Among other things:
- It would make it easier to run locally
- We can centralize where we pick the port numbers (so it's not hard-coded in both the Dockerfile and here)

Reply:
I agree, this is the approach I would like to take. I would like to be able to start and stop the process so we can test things like IPv6, etc. I was hoping to add that in the upcoming PRs or do you prefer it done here?

Reply:
I think IPv6 etc. is definitely out of scope for this PR.

I think making the test itself start/stop the process would be in scope though; the Dockerfile isn't really useful for testing (especially if we pick an unprivileged port).  Ideally we can just start host resolver and have it pick a random port and report what it was… but that might be too much work for this test.